osmosis-1/osmosis.zone_assets.json

```diff
@@ -1672,6 +1672,11 @@
       "path": "transfer/channel-2186/factory/wormhole14ejqjyq8um4p3xfqj74yld5waqljf88fz25yxnma0cngspxe3les00fpjx/5wS2fGojbL9RhGEAeQBdkHPUAciYDxjDTMYvdf9aDn2r",
       "osmosis_main": true,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "fxcore",
+      "base_denom": "FX",
+      "path": "transfer/channel-2716/FX"
```

Comment:
Please add"
      "osmosis_main": false,
      "osmosis_frontier": true

main can be true if the pool has over $2k liquidity

Reply:
pool does not exist yet, but we intend to put over 2k liquidity. Do I set it as true?

Reply:
Sure, as long as it'll grow to a large pool before listing the token, then yes